staff/masha-stepanova/product/app/App.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react'
+
+import { Landing } from './view/custom/Landing.jsx'
+import { Register } from './view/custom/Register.jsx'
+import { Login } from './view/custom/Login.jsx'
+import { Home } from './view/Home/index.jsx'
+import { Profile } from './view/custom/Profile.jsx'
```

Comment:
Profile is a private view and should be managed in the private area (that is, Home).